src/MongoDB.Driver.Core/Core/Misc/ServerVersion.cs

```diff
@@ -0,0 +1,463 @@
+        /// Casts a SemanticVersion to a ServerVersion.
+        /// </summary>
+        /// <param name="semanticVersion">The SemanticVersion.</param>
+        /// <returns>A ServerVersion.</returns>
+        /// <exception cref="FormatException">semanticVersion</exception>
+        public static explicit operator ServerVersion(SemanticVersion semanticVersion)
+        {
+            return Parse(semanticVersion.ToString());
+        }
+
+        /// <summary>
+        /// Determines whether two specified server versions have the same value.
+        /// </summary>
+        /// <param name="a">The first server version to compare, or null.</param>
+        /// <param name="b">The second server version to compare, or null.</param>
+        /// <returns>
+        /// True if the value of a is the same as the value of b; otherwise false.
+        /// </returns>
+        public static bool operator ==(ServerVersion a, ServerVersion b)
+        {
+            if (ReferenceEquals(a, null))
+            {
+                return ReferenceEquals(b, null);
+            }
+
+            return a.CompareTo(b) == 0;
+        }
+
+        /// <summary>
+        /// Determines whether two specified server versions have different values.
+        /// </summary>
+        /// <param name="a">The first server version to compare, or null.</param>
+        /// <param name="b">The second server version to compare, or null.</param>
+        /// <returns>
+        /// True if the value of a is different from the value of b; otherwise false.
+        /// </returns>
+        public static bool operator !=(ServerVersion a, ServerVersion b)
+        {
+            if (ReferenceEquals(a, null))
+            {
+                return !ReferenceEquals(b, null);
+            }
+
+            return a.CompareTo(b) != 0;
+        }
+
+        /// <summary>
+        /// Determines whether the first specified ServerVersion is greater than the second specified ServerVersion.
+        /// </summary>
+        /// <param name="a">The first server version to compare, or null.</param>
+        /// <param name="b">The second server version to compare, or null.</param>
+        /// <returns>
+        /// True if the value of a is greater than b; otherwise false.
+        /// </returns>
+        public static bool operator >(ServerVersion a, ServerVersion b)
+        {
+            if (ReferenceEquals(a, null))
+            {
+                return false;
+            }
+
+            return a.CompareTo(b) > 0;
+        }
+
+        /// <summary>
+        /// Determines whether the first specified ServerVersion is greater than or equal to the second specified ServerVersion.
+        /// </summary>
+        /// <param name="a">The first server version to compare, or null.</param>
+        /// <param name="b">The second server version to compare, or null.</param>
+        /// <returns>
+        /// True if the value of a is greater than or equal to b; otherwise false.
```

Comment:
`;` => `,` ? 
Also, consider: <c>###<c>

Reply:
For context: this text was generated by Ghostdoc.

Reply:
We have `;` in most places, and considering there's no strict guideline on this, I leave it as in most places.

Reply:
>Ghostdoc

@rstam , just out of curiosity, do we use it now? How we can use it if so?

Reply:
GhostDoc is an extension for Visual Studio that automates creating doc comments. It saves a lot of typing.

I just checked and I don't have it installed in Visual Studio 2017, so it looks like Visual Studio 2017 itself now has most of the features of GhostDoc, so I don't think any of us needs to use GhostDoc any more.

I mentioned GhostDoc because that format you found odd for the \<returns> section was not something that anyone went out of the way to write, it was just the way GhostDoc handled boolean return values.

Reply:
@rstam  Thanks for your explanation. I think it will be worth trying to standardize it in the google doc